spec/features/dams_collections_spec.rb

```diff
@@ -148,4 +155,28 @@
     expect(page).to have_selector("#collections-image img")
   end
 
+end
+
+feature "Visitor wants to view a collection's page" do
+
+  before(:all) do
+    @part = DamsProvenanceCollectionPart.create titleValue: 'Sample Provenance Part', visibility: 'curator'
+    @prov = DamsProvenanceCollection.create titleValue: 'Sample Provenance Collection', provenanceCollectionPartURI: @part.pid, visibility: 'curator'
+    @part.provenanceCollectionURI = @prov.pid
+    @part.save
+    solr_index @prov.pid
+    solr_index @part.pid
+  end
+
+  after(:all) do
+    @prov.delete
+    @part.delete
+  end
+
+  scenario 'should see access control information (curator view)' do
+    sign_in_developer
+    visit dams_collection_path @prov.pid
+    expect(page).to have_content('AccessCurator Only')
```

Comment:
No. It displays as "AccessCurator Only" in rspec because I am using <span> elements to style them in regular browsers.